src/mod2c_core/portability.h

```diff
@@ -0,0 +1,13 @@
+#ifndef mod2c_portability_h
+#define mod2c_portability_h
+
+#if (defined(_WIN32) || defined(__WIN32__))
+// mkdir doesn't accept permission parameter on mingw
+#define mkdir(directory, permission) mkdir(directory)
```

Comment:
I've not seed this kind of define before. Does every compiler (preprocessor) understand it?

Reply:
> Does every compiler (preprocessor) understand it?

I was thinking of more gnu-mingw toolchain. Visual studio defines WIN32 which is different.

In the latest commit I explicitly defined MINGW (and removed some old code which is not relevant anymore for mod2c).